client/docs/providers-and-renderers.md

```diff
@@ -1,3 +1,7 @@
+**Notice** Consider using [Composables](composalbes.md) instead of Providers. They offer more functionality and need less boilerplate.
```

Comment:
```suggestion
**Notice** Consider using [Composables](composables.md) instead of Providers. They offer more functionality and need less boilerplate.
```